src/locales/fr/common.json

```diff
@@ -3466,7 +3466,6 @@
   "Terms": {
     "title": "Conditions d'utilisation",
     "read": "Lire les Conditions d'utilisation",
-    "switchLabelPrivacy": "J'ai lu et j'accepte la <1>Politique de confidentialité</1>.",
```

Comment:
note for future: please never touch the src/locales/* generated files. it creates conflicts and it's also not necessary because crowdin is going to remove this for you :)
we only work on the /en